test/next_ls_test.exs

```diff
@@ -888,6 +888,106 @@ defmodule NextLSTest do
     end
   end
 
+  describe "find references" do
+    @describetag root_paths: ["my_proj"]
+    setup %{tmp_dir: tmp_dir} do
+      File.mkdir_p!(Path.join(tmp_dir, "my_proj/lib"))
+      File.write!(Path.join(tmp_dir, "my_proj/mix.exs"), mix_exs())
+      [cwd: tmp_dir]
+    end
+
+    setup %{cwd: cwd} do
+      peace = Path.join(cwd, "my_proj/lib/peace.ex")
+
+      File.write!(peace, """
+      defmodule MyApp.Peace do
+        def and_love() do
+          "✌️"
+        end
+      end
+      """)
+
+      bar = Path.join(cwd, "my_proj/lib/bar.ex")
+
+      File.write!(bar, """
+      defmodule Bar do
+        alias MyApp.Peace
+        def run() do
+          Peace.and_love()
+        end
+      end
+      """)
+
+      [bar: bar, peace: peace]
+    end
+
+    setup :with_lsp
+
+    test "list function references", %{client: client, bar: bar, peace: peace} do
+      assert :ok == notify(client, %{method: "initialized", jsonrpc: "2.0", params: %{}})
+      assert_request(client, "client/registerCapability", fn _params -> nil end)
+      assert_notification "window/logMessage", %{"message" => "[NextLS] Runtime for folder my_proj is ready..."}
+      assert_notification "window/logMessage", %{"message" => "[NextLS] Compiled!"}
+
+      request(client, %{
+        method: "textDocument/references",
+        id: 4,
+        jsonrpc: "2.0",
+        params: %{
+          position: %{line: 1, character: 6},
+          textDocument: %{uri: uri(peace)},
+          context: %{includeDeclaration: true}
+        }
+      })
+
+      uri = uri(bar)
+
+      assert_result 4,
+                    [
+                      %{
+                        "uri" => ^uri,
+                        "range" => %{
+                          "start" => %{"line" => 3, "character" => 10},
+                          "end" => %{"line" => 3, "character" => 18}
+                        }
+                      }
+                    ],
+                    500
```

Comment:
I have the timeout for these globally set to 30 seconds, so you can remove all of these custom timeouts, as it's actually making it shorter 😂

https://github.com/elixir-tools/next-ls/blob/main/test/test_helper.exs#L9

Reply:
I've just pushed a commit that removes those explicit timeouts.

Reply:
🚀